.github/workflows/check_deps.yml

```diff
@@ -0,0 +1,68 @@
+---
+
+name: Check Dependencies
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '38 8 * * *'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions: write-all
+
+jobs:
+  check-deps:
+    name: Check Dependencies
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: '27'
+          rebar3-version: '3.24'
+          version-type: 'strict'
```

Comment:
I believe `strict` won't pick up minor and patch releases but will strictly use 27.0 for example. Don't we want to allow for any 27.x maybe?

Reply:
You are right. That also applied to rebar. I've relaxed versions, starting from https://github.com/dnsimple/erldns-zoneclient/actions/runs/13895126993/job/38873991068 that was already merged.

Reply:
Addressed in https://github.com/dnsimple/bugsnag-erlang/pull/40/commits/734feec15edd652aa350d52b497d9198f892cf3f